### PR TITLE
Admin nav update

### PR DESCRIPTION
**Pull request by alejandroereyes**
updated tabs and moved nav into a partial

### CODE REVIEW

File: app/controllers/meetings_controller.rb [hunk 1]
```diff
@@ -17,9 +17,11 @@ def index
...
   def show
     @meeting = Meeting.find(params[:id])
-    # if user_is_admin? && @meeting.room.company.id == current_employee.company.id ||
-    #    @meeting.invitations.exists?(employee_id: current_employee.id) ||
-    #    @meeting.employee_id == current_employee.id
+    # render json: {meeting: @meeting, e_com: current_employee.company.id, met_comp: @meeting.employee.company.id}
+    # if current_employee.company.id == @meeting.employee.company.id # is user part of the company
+    # # if user_is_admin? && @meeting.room.company.id == current_employee.company.id ||
+       # @meeting.invitations.exists?(employee_id: current_employee.id) ||
+       # @meeting.employee_id == current_employee.id
```
Comment (jah2488) [inline app/controllers/meetings_controller.rb:24]: Why _more_ comments?

File: app/views/admin/_adnav.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ <ul class="nav nav-tabs">
+    <li role="presentation" class=<%= 'active' if request.fullpath == dashboard_path %>><%= link_to 'Admin Dashboard', dashboard_path %></li>
+    <li role="presentation" class=<%= 'active' if request.fullpath == new_room_path %>><a href="/rooms/new">Create a Room</a></li>
+    <li role="presentation" class=<%= 'active' if request.fullpath == admin_reports_meetings_path %>><a href="/admin/reports_meetings">Meetings Report</a></li>
+    <li role="presentation" class=<%= 'active' if request.fullpath == admin_reports_rooms_path %>><a href="/admin/reports_rooms">Rooms Report</a></li>
+    <li role="presentation" class=<%= 'active' if request.fullpath == admin_busiest_employees_path %>><a href="/admin/busiest_employees">Employee Report</a></li>
+    <li role="presentation" class=<%= 'active' if request.fullpath == admin_add_branding_path %>><a href="/admin/add_branding">Branding</a></li>
+  </ul>
```
Comment (jah2488) [inline app/views/admin/_adnav.html.erb:8]: Nice and tidy, but I wonder if this code could be _abstracted_ into a method inside `application_helper`?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -50,12 +50,7 @@
   </div><!-- /.container-fluid -->
 </nav>
 <% if user_is_admin? %>
-  <ul class="nav nav-tabs">
-    <li role="presentation" class="active"><%= link_to 'Admin Dashboard', dashboard_path %></li>
-    <li role="presentation"><a href="/rooms/new">Create a Room</a></li>
-    <li role="presentation"><a href="/meetings/new">Create a new Meeting</a></li>
-    <li role="presentation"><a href="/admin">Reports</a></li>
-  </ul>
+  <%= render partial: 'admin/adnav' %>
```
Comment (jah2488) [inline app/views/layouts/application.html.erb:53]: Yay!

**Comment by JMD1986** [comment]
:fire: :fire: :fire:

**Comment by mlgiardina** [comment]
:+1:

**Comment by mikepitre** [comment]
:+1: :fire:

**Comment by mlgiardina** [comment]
:fire_engine: :fire_engine: :fire_engine:  putting out those fires

**Pull request merged**